### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ivam2280**
- [DEMO LINK](https://Ivam2280.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Ivam2280.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,12 +11,113 @@
...
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ol class="nav__point">
+          <li class="nav__item apple">
```
Comment (anastasiiavorobiova) [inline src/index.html:47]:
```suggestion
          <li class="nav__item">
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+  margin-left: 20px;
+}
+
+.logo {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+}
+
+.apple {
```
Comment (anastasiiavorobiova) [inline src/style.css:35]: Consider using `:first-child`

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  width: 37px;
```
Comment (anastasiiavorobiova) [inline src/style.css:62]:
```suggestion
  width: 100%;
```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

File: src/style.css (hunk 3 above)
Comment (Ivam2280) [reply src/style.css]: Thank you for your support!

**Pull request closed without merging**